app/src/main/java/com/duckduckgo/app/browser/WebDataManager.kt

```diff
@@ -146,7 +149,7 @@ class WebViewDataManager @Inject constructor(
             }.onSuccess {
                 excludedDirectories.add("IndexedDB")
             }.onFailure { t ->
-                Timber.w(t, "Failed to clear IndexedDB, will delete it instead")
+                logcat(WARN) { "Failed to clear IndexedDB, will delete it instead" + ": " + t.asLog() }
```

Comment:
`logcat(WARN) { "Failed to clear IndexedDB, will delete it instead: ${t.asLog()}" }`